File: 卡码/KamaCoder/Go/test03.go
//3. A+B问题III
package main

import (
	"fmt"
)

func main(){
	for{
		var a,b int
		fmt.Scanln(&a,&b)
		if a==0&&b==0 {
			return
		}
		fmt.Println(a+b)
	}
}

File: 卡码/KamaCoder/C++/test03.cpp
//3. A+B问题III
#include <iostream>
using namespace std;

int main(){
	int a,b;
	while(scanf("%d %d\n",&a,&b)!=EOF){
		if(a==0&&b==0) break;
		cout<<a+b<<endl;
	}
	return 0;
}